src/pages/index.js

```diff
@@ -263,6 +263,29 @@ export default class Index extends React.Component {
       )
     }
 
+    const Awards = () => {
+      return (
+        <div className="awardsSection paddingBottom paddingTop">
+          <h2>Awards</h2>
+          <Block layout="threeColumn" background={null}>
+            {[
+              {
+                image: `${baseUrl}img/impactful-conribution-award-399x544.png`,
+                imageAlign: 'top',
+                imageLink:
+                  'https://user-images.githubusercontent.com/1500684/106369944-ece62900-6312-11eb-9bbf-47bba0bf0d6f.png',
```

Comment:
```suggestion
                imageLink: 'https://osawards.com/react/2019',
```
This is meant as a link to the source of that award not a link to the image of that award